thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -167,39 +172,40 @@
       {{!-- Alert anomaly table --}}
       <div class="te-block-container">
         {{#if baselineOptionsLoading}}
-          <div class="spinner-wrapper spinner-wrapper--card">{{ember-spinner}}</div>
+          <div class="spinner-wrapper-self-serve spinner-wrapper-self-serve__custom">{{ember-spinner}}</div>
         {{/if}}
         <table class="te-anomaly-table">
```

Comment:
I'm prepared to refactor this table in one of the next iterations to nicely "componentize" it.